terra/docs/testing.md

```diff
@@ -0,0 +1,32 @@
+# Wormhole + Terra local test environment
+
+The following dependencies are required for local development:
+
+- [Go](https://golang.org/dl/) >= 1.15.3
+- [Docker](https://docs.docker.com/engine/install/) / moby-engine >= 19.03
+- [Tilt](http://tilt.dev/) >= 0.17.2
+- Any of the local Kubernetes clusters supported by Tilt. 
+  We recommend [minikube](https://kubernetes.io/docs/setup/learning-environment/minikube/) with the kvm2 driver.
+- [Node.js](https://nodejs.org/) >= 14.x, [ts-node](https://www.npmjs.com/package/ts-node) >= 8.x
```

Comment:
We should not require application dependencies on the host - better to move this to a container, either running in k8s or similar to how we use abigen.